### PR TITLE
feat: netwatch binary sensor

### PR DESCRIPTION
**Pull request by L2jLiga**
## Proposed change

Added support for tracking Netwatch as binary sensor.

Related issue #289 

## Type of change

- [ ] Bugfix
- [X] New feature
- [ ] Code quality improvements to existing code or addition of tests
- [ ] Documentation

## Checklist

- [X] The code change is tested and works locally.
- [X] The code has been formatted using Black.
- [ ] Tests have been added to verify that the new code works.
- [X] Documentation added/updated if required.

### CODE REVIEW

File: custom_components/mikrotik_router/binary_sensor.py [hunk 1]
```diff
@@ -145,3 +149,38 @@ def extra_state_attributes(self) -> Mapping[str, Any]:
...
+        """Return the icon."""
+        icon = super(MikrotikNetwatchBinarySensor, self).icon
+
+        if not self._data["enabled"]:
+            icon = "mdi:lan-disconnect"
+
+        return icon
+
+    @property
+    def extra_state_attributes(self) -> Mapping[str, Any]:
```
Comment (tomaae) [inline custom_components/mikrotik_router/binary_sensor.py:178]: This part is redundant, it is already set up in entity description

File: custom_components/mikrotik_router/binary_sensor_types.py [hunk 2]
```diff
@@ -177,6 +186,24 @@ class MikrotikBinarySensorEntityDescription(BinarySensorEntityDescription):
...
+        ha_connection=DOMAIN,
+        ha_connection_value="Netwatch",
+        data_path="netwatch",
+        data_attribute="status",
+        data_name="host",
+        data_name_comment=True,
+        data_uid="host",
+        data_reference="host",
+        data_attributes_list=DEVICE_ATTRIBUTES_NETWATCH,
+        func="MikrotikNetwatchBinarySensor",
```
Comment (tomaae) [inline custom_components/mikrotik_router/binary_sensor_types.py:205]: We can probably just use MikrotikBinarySensor

File: custom_components/mikrotik_router/entity.py [hunk 3]
```diff
@@ -70,6 +72,14 @@ def _skip_sensor(config_entry, entity_description, data, uid) -> bool:
     ):
         return True
 
+    if (
+        entity_description.func == "MikrotikNetwatchBinarySensor"
```
Comment (tomaae) [inline custom_components/mikrotik_router/entity.py:76]: Change to `entity_description.data_path == "netwatch"`

File: custom_components/mikrotik_router/translations/en.json
Comment (tomaae) [inline custom_components/mikrotik_router/translations/en.json:1]: translation files should not be edited directly, add entries into strings.json

File: custom_components/mikrotik_router/translations/ru.json
Comment (tomaae) [inline custom_components/mikrotik_router/translations/ru.json:1]: translation files should not be edited directly, add entries into strings.json

File: custom_components/mikrotik_router/translations/uk.json
Comment (tomaae) [inline custom_components/mikrotik_router/translations/uk.json:1]: translation files should not be edited directly, add entries into strings.json

File: custom_components/mikrotik_router/binary_sensor.py [hunk 4]
```diff
@@ -145,3 +149,38 @@ def extra_state_attributes(self) -> Mapping[str, Any]:
                     attributes[format_attribute(variable)] = self._data[variable]
 
         return attributes
+
+
+# ---------------------------
+#   MikrotikNetwatchBinarySensor
+# ---------------------------
+class MikrotikNetwatchBinarySensor(MikrotikBinarySensor):
```
Comment (tomaae) [inline custom_components/mikrotik_router/binary_sensor.py:157]: Separate class for this sensor is most likely not needed

File: custom_components/mikrotik_router/binary_sensor.py [hunk 5]
```diff
@@ -42,6 +45,7 @@ async def async_setup_entry(
         "MikrotikBinarySensor": MikrotikBinarySensor,
         "MikrotikPPPSecretBinarySensor": MikrotikPPPSecretBinarySensor,
         "MikrotikPortBinarySensor": MikrotikPortBinarySensor,
+        "MikrotikNetwatchBinarySensor": MikrotikNetwatchBinarySensor,
```
Comment (tomaae) [inline custom_components/mikrotik_router/binary_sensor.py:48]: Separate class for this sensor is most likely not needed

**Comment by tomaae** [comment]
Nice work, just few minor changes are needed to remove redundant code.

**Comment by L2jLiga** [comment]
Thanks for review, I've updated PR

**Pull request merged**